aiosmtpd/tests/test_smtp.py

```diff
@@ -434,6 +434,13 @@ def test_mail_params_bad_syntax_esmtp(self):
                 response,
                 b'Syntax: MAIL FROM: <address> [SP <mail-parameters>]')
 
+    def test_mail_fail_parse_email(self):
+        with SMTP(*self.address) as client:
+            client.helo('example.com')
+            code, response = client.docmd('MAIL FROM: <""@example.com>')
```

Comment:
Won't this fail to fail when [bpo-27931](http://bugs.python.org/issue27931) is fixed?  Can you change the example to something that will always be considered invalid?